content/en/docs/setup/install/multicluster/shared/index.md

```diff
@@ -208,7 +208,7 @@ EOF
 Apply the main cluster's configuration.
 
 {{< text bash >}}
-$ istioctl --context=${MAIN_CLUSTER_CTX} manifest apply -f istio-main-cluster.yaml
+$ istioctl --context=${MAIN_CLUSTER_CTX} install -f istio-main-cluster.yaml
```

Comment:
it's a good time to fix the ordering here

Reply:
There are other related changes and I will create separate PR to fix that because it's not related to `istioctl install`. Thanks for your suggestion.